### ISSUE TITLE
The mini-followup daxes have invalid PFNs

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#668]
The mini-followup workflows are generating DAXes with invalid PFNs. This is ignored by Pegasus 4.5.x, but is flagged as an error in 4.6.x. For example the lines
```xml
<file name="L1-HDF_TRIGGER_MERGE_FULL_DATA-1133173817-1276200.hdf">
      <pfn url="L1-HDF_TRIGGER_MERGE_FULL_DATA-1133173817-1276200.hdf" site="local"/>
</file>
```
are invalid and generate the error:
```
2015.12.19 13:03:09.491 EST: [FATAL ERROR]  java.lang.RuntimeException: Unable to parse URL file:/
```
The DAX works in 4.5.x because a valid LFN to PFN map is contained in the cache file passes to ```pegasus-plan```. The easiest solution is just to omit the invalid PFN from the DAX.

**Comment by ahnitz** [comment]
It is just not a full path, no? I would think the better solution would be to check for that and convert to an absolute path if it appears relative. Otherwise, we've hard-coded an inability to run any of the minifollowups outside of a larger workflow. They should be able to be run stand-a-lone.

**Comment by duncan-brown** [comment]
A full path would work. This path may be incorrect, but can be overridden by the cache file passed to ```pegasus-plan```.

**Comment by ahnitz** [comment]
Yeah, as long as the cache overrides the internal one, it should be OK, and
keeps the ability to run the workflow by itself. I believe we already
include the necessary pegasus config option so that happens.

**Comment by duncan-brown** [comment]
@ahnitz any chance you could take a quick look at this? You might get to it before Ian or I do and it's a huge pain to downgrade Pegasus from 4.6 to 4.5, unfortunately.

**Comment by ahnitz** [comment]
I think you've merged the related pull request #669, so I am closing this issue.
